recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+    
+
+build:
+    number: 0
+    script: $PYTHON setup.py install
+    skip: True # [win and not py27]
+
+requirements:
+  build:
+    - python
+    - python >=2.7
```

Comment:
Moreover, there seems to be a conda bug right now that might give you unsatisfiable dependencies when python has a version spec.  Best to avoid it for now.